src/data/faq.json

```diff
@@ -467,20 +467,26 @@
                                 "anchor": "eu_dcc_export",
                                 "active": false,
                                 "textblock": [
+                                    "<b>Updated ??? ?? ????</b><hr>",
                                     "<b>Purpose</b>",
                                     "As of version 2.10 of the Corona-Warn-App, you can export scanned EU Digital COVID Certificates into PDF documents. This function is primarily used to create backup copies of the certificates for your personal purposes.",
+                                    "As of version 2.24 of the Corona-Warn-App, you can save all vaccination and recovery certificates available in your app, as well as all tests that are not older than 72 hours, in a single PDF document.",
                                     "Of course, the original EU templates are used for the layout of the certificates.",

```

Comment:
Suggest to remove "Of course" from this sentence as in English there is an undertone that the reader may be a bit stupid for thinking otherwise. It does not add to the information value of the sentence.

"Of course, the" => "The"

Reply:
Fixed in https://github.com/corona-warn-app/cwa-website/pull/2961/commits/7e085cdd9b7b63f28941b34fb0eacc970ebf8d85, thanks for your suggestion